apps/emqx/src/emqx_persistent_session_ds_sup.erl

```diff
@@ -55,7 +55,8 @@ do_init(_Opts) ->
     },
     CoreChildren = [
         worker(session_gc_worker, emqx_persistent_session_ds_gc_worker, []),
-        worker(message_gc_worker, emqx_persistent_message_ds_gc_worker, [])
+        worker(message_gc_worker, emqx_persistent_message_ds_gc_worker, []),
+        worker(persistent_message_cache, emqx_ds_cache, [emqx_persistent_message:cache_opts()])
```

Comment:
Should we spawn this under `emqx_ds_builtin_db_sup`?

Reply:
Yes, I think it's a good idea. Also, I don't see why don't replicant have the cache?